src-vue/src/views/ThunderstoreModsView.vue

```diff
@@ -42,68 +50,55 @@ export default defineComponent({
         mods(): ThunderstoreMod[] {
             return this.$store.state.thunderstoreMods;
         },
+        filteredMods(): ThunderstoreMod[] {
+            if (this.searchValue.length === 0) {
+                return this.mods;
+            }
+
+            return this.mods.filter((mod: ThunderstoreMod) => {
+                return mod.name.toLowerCase().includes(this.searchValue)
+                    || mod.owner.toLowerCase().includes(this.searchValue)
+                    || mod.versions[0].description.toLowerCase().includes(this.searchValue);
+            });
+        },
         modsList(): ThunderstoreMod[] {
-            return this.input.length === 0 || this.userIsTyping ? this.mods : this.filteredMods;
+            return this.input.length !== 0 || this.userIsTyping ? this.filteredMods : this.mods;
+        },
+        modsPerPage(): number {
+            return +this.$store.state.mods_per_page;
```

Comment:
From testing, `mods_per_page` was sometimes stored as a string, so the `+` casts it into a number.
No clue if it's useful though :)

Reply:
I'd suggest making the cast explicit then. I assume TypeScript has something like `int()` in Python. If not (which would surprise me) I'd add a comment stating that why the `+` is there to avoid a "_Remove unknown `+`_" kind of commit in the future ^^"

Reply:
A JavaScript dev will never tell you to "Remove unknown `+`" 😛 
https://stackoverflow.com/a/14668510/11243782

Reply:
Given that the value will always ever be sth like `23` or `"23"` I think we should just use `parseInt()` in this case ^^

Reply:
Addressed in b618796fc2cd1268e7cec53fa386bdc1598cea6a